### ISSUE TITLE
[TF2] Projectiles randomly hit or miss teammates

### ISSUE THREAD

**Issue by ldesgoui** [ValveSoftware/Source-1-Games#2491]
Hello,

Projectiles such as the Rockets and Grenades randomly pass through or explode upon contact with teammates. The preferred default should be passing through.

Thanks

**Comment by Exaeta** [comment]
@ldesgoui I think there's a minimum distance at which they impact your team mates, before then they just pass through.

**Comment by ldesgoui** [comment]
You're right, rockets will always hit a teammate if the rocket has traveled for longer than a certain amount (.5s?). This feels extremely random during gameplay, but I guess it's not a bug. @kisak-valve could you add "Feature request"? Thanks

**Comment by JoinedSenses** [comment]
Yeah, this is a feature, not a bug. If you're looking for a plugin which allows rockets/pipes to pass through teammates for a custom server, get in touch with me on steam under the same name. Ive got one at the moment.
